ephios/plugins/basesignup/signup/simple_qualification_required.py

```diff
@@ -0,0 +1,69 @@
+import typing
+
+from django import forms
+from django.utils.translation import gettext_lazy as _
+from django.utils.translation import pgettext_lazy
+
+from ephios.core.models import AbstractParticipation
+from ephios.core.signup.methods import BaseSignupMethod
+from ephios.plugins.basesignup.signup.common import QualificationMinMaxBaseSignupMethod
+
+_Base = BaseSignupMethod if typing.TYPE_CHECKING else object
+
+
+class AutomaticConfirmationMixin(_Base):
+    """mixin for signup methods where an instant confirmation can be optionally enabled"""
+
+    @property
+    def configuration_form_class(self):
+        class ConfigurationForm(super().configuration_form_class):
+            instant_confirmation = forms.BooleanField(
+                required=False,
+                initial=False,
+                label=_("Instant confirmation"),
+                help_text=_("Instantly confirm every signup (no disposition required)"),
+            )
+
+        return ConfigurationForm
+
+    def _configure_participation(
+        self, participation: AbstractParticipation, **kwargs
+    ) -> AbstractParticipation:
+        if self.configuration.instant_confirmation:
+            participation.state = AbstractParticipation.States.CONFIRMED
+        else:
+            participation.state = AbstractParticipation.States.REQUESTED
+        return participation
+
+    @property
+    def signup_view_class(self):
+        if not self.configuration.instant_confirmation:
+
+            class SignupView(super().signup_view_class):
+                signup_success_message = _(
+                    "You have successfully requested a participation for {shift}."
+                )
+                signup_error_message = _("Requesting a participation failed: {error}")
+
+            return SignupView
+        return super().signup_view_class
+
+    @property
+    def registration_button_text(self):
+        if self.configuration.instant_confirmation:
+            return super().registration_button_text
+        return pgettext_lazy("signup button text", "Request")
+
+    @property
+    def uses_requested_state(self):
+        return not self.configuration.instant_confirmation
+
+
+class SimpleQualificationRequiredSignupMethod(
+    AutomaticConfirmationMixin, QualificationMinMaxBaseSignupMethod
+):
+    slug = "simple_qualification_required"
+    verbose_name = _("Signup where a qualification can be required")
```

Comment:
@jeriox what do you think of this name? I think this style of phrasing would allow users to better differentiate between the couple/no sign up methods as well as between the actual signup methods. It also shows that the qualification requirement is optional

Reply:
Hm, I'd like to see all of the names next to each other to judge that. My first thought was that starting the name with "signup" is a bit redundant since the field is already labeled with signup method. 
I still like the idea of using the planners POV since they need to grasp the concept...

Reply:
We can change the label on the form field though. Or do some custom templating entirely.

Reply:
I think we should review the names when section-based and complex is finished.